### PR TITLE
mark `vela install` as deprecated

### LINKED ISSUE #978: mark `vela install` as deprecated
the `cobra` has a deprecate flag, we should add a message to say:
  > we will remove this command after `Kubevela 1.0`.

And refactor all related message to use `helm install` instead:

image

part of #942

### PR DESCRIPTION
**Pull request by wangkai1994**
To fix #978

image

### CODE REVIEW

**Comment by wangkai1994** [comment]
so how to retest ?

**Comment by wonderflow** [comment]
Thanks for your contribution, please also fix other places which contain `vela install` info.

File: pkg/commands/dashboard.go [hunk 1]
```diff
@@ -224,8 +224,9 @@ func CheckVelaRuntimeInstalledAndReady(ioStreams cmdutil.IOStreams, c client.Cli
 		ioStreams.Info(fmt.Sprintf("\n%s %s%s or %s",
 			emojiLightBulb,
 			"Please use this command to install: ",
-			white.Sprint("vela install -w"),
-			white.Sprint("vela install --help")))
+			white.Sprint("helm repo add kubevela https://kubevelacharts.oss-cn-hangzhou.aliyuncs.com/core && helm repo update"),
+			white.Sprint("kubectl create namespace vela-system"),
+			white.Sprint("helm install -n vela-system kubevela kubevela/vela-core")))
```
Comment (wonderflow) [inline pkg/commands/dashboard.go:229]: good job

File: pkg/commands/system.go [hunk 2]
```diff
@@ -123,6 +123,7 @@ func NewInstallCommand(c types.Args, chartContent string, ioStreams cmdutil.IOSt
 		Annotations: map[string]string{
 			types.TagCommandType: types.TypeStart,
 		},
+		Deprecated: "This flag is DEPRECATED and we will remove this command after Kubevela 1.0.Use helm instead",
```
Comment (wonderflow) [inline pkg/commands/system.go:126]:
```suggestion
		Deprecated: "vela install is DEPRECATED and we will remove it after Kubevela 1.0. Please use helm chart instead",
```

File: pkg/commands/system.go [hunk 3]
```diff
@@ -282,7 +283,7 @@ func GetOAMReleaseVersion(ns string) (string, error) {
 			return result.Chart.AppVersion(), nil
 		}
 	}
-	return "", errors.New("kubevela not found in your kubernetes cluster, try `vela install` to install")
+	return "", errors.New("oam-kubernetes-runtime not found in your kubernetes cluster, try `helm install -n vela-system kubevela kubevela/vela-core` to install")
```
Comment (wonderflow) [inline pkg/commands/system.go:286]:
```suggestion
	return "", errors.New("kubevela chart not found in your kubernetes cluster,  refer to 'https://kubevela.io/#/en/install' for installation")
```

File: e2e/raw-objects/samples/rollout-demo/README.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ helm install -n flagger rollout charts/flagger/
 ## Install Vela core
 ```shell script
 make
-bin/vela install
+helm install -n vela-system kubevela charts/vela-core
```
Comment (wonderflow) [inline e2e/raw-objects/samples/rollout-demo/README.md:14]: please leave it as it is. Or complete the whole instruction:

1. add helm repo
2. update repo

File: docs/examples/kubecondemo/README.md [hunk 5]
```diff
@@ -9,7 +9,7 @@
 * Install Crossplane(later)
 * Download KubeVela release from [release page](https://github.com/oam-dev/kubevela/releases/tag/v0.0.9)
 * Unpack the package and add it to `PATH` by running `sudo mv ./vela /usr/local/bin/vela`
-* Run `vela install`
+* Run `helm install`
```
Comment (wonderflow) [inline docs/examples/kubecondemo/README.md:12]: please don't change here

File: e2e/raw-objects/samples/rollout-demo/README.md (hunk 4 above)
Comment (wangkai1994) [reply e2e/raw-objects/samples/rollout-demo/README.md:14]: I agree

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -74,7 +74,7 @@ make core-run
 This command will run controller locally, it will use your local KubeConfig which means you need to have a k8s cluster
 locally. If you don't have a one, we suggest that you could setup up a cluster with [kind](https://kind.sigs.k8s.io/).
 
-When you're developing `vela-core`, make sure the controller installed by `vela install` is not running.
+When you're developing `vela-core`, make sure the controller installed by `helm install` is not running.
```
Comment (wonderflow) [inline CONTRIBUTING.md:77]:
```suggestion
When you're developing `vela-core`, make sure the controller installed by helm chart is not running.
```

File: docs/en/_sidebar.md [hunk 7]
```diff
@@ -51,7 +51,6 @@
       - [vela config](/en/cli/vela_config.md)
       - [vela env](/en/cli/vela_env.md)
       - [vela init](/en/cli/vela_init.md)
-      - [vela install](/en/cli/vela_install.md)
```
Comment (wonderflow) [inline docs/en/_sidebar.md:54]: should not delete it

**Review by wonderflow — APPROVED** [review]
Good Job！

**Pull request merged**